app/assets/stylesheets/components/multi-input.scss

```diff
@@ -50,4 +50,33 @@
     background: none;
     color: var(--accent-danger);
   }
+
+  &-invalid {
+    &:first-child,
+    &:last-child {
+      background-color: var(--accent-danger-a10);
+      color: var(--accent-danger);
+      border: solid var(--accent-danger);
```

Comment:
For some reason the borders are showing up really thick for me - I'm going to add in a specific border-width value to these to make sure it always looks like your screenshot above 👍